bakery/src/scripts/canonical_book_list.json

```diff
@@ -0,0 +1,104 @@
+{
+    "canonical_books": [
+        {
+            "uuid": "02040312-72c8-441e-a685-20e9333f3e1d",
+            "_name": "Introduction to Sociology 2e"
+        },
+        {
+            "uuid": "02776133-d49d-49cb-bfaa-67c7f61b25a1",
+            "_name": "Intermediate Algebra"
+        },
+        {
+            "uuid": "031da8d3-b525-429c-80cf-6c8ed997733a",
+            "_name": "College Physics"
+        },
+        {
+            "uuid": "14fb4ad7-39a1-4eee-ab6e-3ef2482e3e22",
+            "_name": "Anatomy and Physiology"
+        },
+        {
+            "uuid": "1d39a348-071f-4537-85b6-c98912458c3c",
+            "_name": "Calculus Volume 2"
+        },
+        {
+            "uuid": "2e737be8-ea65-48c3-aa0a-9f35b4c6a966",
+            "_name": "Astronomy"
+        },
+        {
+            "uuid": "30189442-6998-4686-ac05-ed152b91b9de",
+            "_name": "Introductory Statistics"
+        },
+        {
+            "uuid": "4abf04bf-93a0-45c3-9cbc-2cefd46e68cc",
+            "_name": "Psychology"
+        },
+        {
+            "uuid": "4e09771f-a8aa-40ce-9063-aa58cc24e77f",
+            "_name": "Introduction to Business"
+        },
+        {
+            "uuid": "7a0f9770-1c44-4acd-9920-1cd9a99f2a1e",
+            "_name": "University Physics Volume 2"
+        },
+        {
+            "uuid": "7fccc9cf-9b71-44f6-800b-f9457fd64335",
+            "_name": "Chemistry 2e "
+        },
+        {
+            "uuid": "8b89d172-2927-466f-8661-01abc7ccdba4",
+            "_name": "Calculus Volume 1"
+        },
+        {
+            "uuid": "8d50a0af-948b-4204-a71d-4826cba765b8",
+            "_name": "Biology 2e"
+        },
+        {
+            "uuid": "914ac66e-e1ec-486d-8a9c-97b0f7a99774",
+            "_name": "Business Ethics"
+        },
+        {
+            "uuid": "920d1c8a-606c-4888-bfd4-d1ee27ce1795",
+            "_name": "Principles of Accounting, Volume 2: Managerial Accounting"
+        },
+        {
+            "uuid": "9ab4ba6d-1e48-486d-a2de-38ae1617ca84",
+            "_name": "Principles of Accounting, Volume 1: Financial Accounting"
+        },
+        {
+            "uuid": "9b08c294-057f-4201-9f48-5d6ad992740d",
+            "_name": "College Algebra"
```

Comment:
Based on recent discussions, I believe we're missing Precalculus / `fd53eae1-fa23-47c7-bb1b-972349835c3c` from this list of books.